app/controllers/request-to-go-live/organisation-address/get.controller.test.js

```diff
@@ -73,6 +74,11 @@ describe('organisation address get controller', () => {
         expect(responseData.args[2]).to.equal('request-to-go-live/organisation-address')
 
         const pageData = responseData.args[3]
+        expect(pageData.isRequestToGoLive).to.equal(true)
```

Comment:
Looks like lines 77-78 are duplicates of lines 82-83

Reply:
Good spot